log_outgoing_requests/models.py

```diff
@@ -174,6 +174,15 @@ def response_body_decoded(self) -> str:
 
     response_body_decoded.short_description = _("Response body")  # type: ignore
 
+    @cached_property
+    def response_content_length(self) -> str:
+        """
+        Get Response content length by reading `len(body)`.
+        """
+        return str(len(self.response_body_decoded))
```

Comment:
Is there a specific reason why this is cast to a string? I feel like it would be better to leave it as an `int`, since that is the appropriate type for content length